config/jobs/kubernetes/release/release-config.yaml

```diff
@@ -60,8 +60,38 @@ presubmits:
       testgrid-tab-name: release-test
       testgrid-alert-email: release-managers@kubernetes.io
       testgrid-num-columns-recent: '30'
-# package tests
 periodics:
+- interval: 2h
```

Comment:
2h seems aggressive.

Is there anybody from @kubernetes/sig-testing-pr-reviews who might be able to help quantify cost impact of this?

Reply:
@tpepper -- 2h is so I can get some initial signal out of this; we can potentially increase the interval later.
Impact-wise, it should be negligible, as all this is doing is executing a `./gcbmgr` command, which means the stage will happen on GCB, not Prow.